packages/MSBot/src/msbot-clone-services.ts

```diff
@@ -250,7 +250,7 @@ async function processConfiguration(): Promise<void> {
         // verify az command exists and is correct version
         await checkAzBotServiceVersion();
 
-        command = `az account show`;
+        command = `az account show `;
```

Comment:
Please add a comment here as to why there is a space. 

12 months from now, I would see this code and remove the space as part of general housekeeping.